lib/FreeRTOS-Plus-TCP/source/FreeRTOS_DNS.c

```diff
@@ -567,7 +568,7 @@ TickType_t xWriteTimeOut_ms = ipconfigSOCK_DEFAULT_SEND_BLOCK_TIME;
 					if( lBytes > 0 )
 					{
 						/* The reply was received.  Process it. */
-						ulIPAddress = prvParseDNSReply( pucUDPPayloadBuffer, lBytes, xIdentifier );
+						ulIPAddress = prvParseDNSReply( pucUDPPayloadBuffer, ( size_t )lBytes, xIdentifier );
```

Comment:
Since the receive routine returns int32_t, would it make more sense to change prvParseDNSReply to take that type instead of size_t?